repair/repair.cc

```diff
@@ -1529,6 +1533,10 @@ future<> sync_data_using_repair(seastar::sharded<database>& db,
         rlogger.info("repair id {} to sync data for keyspace={}, status=started", id, keyspace);
         return repair_tracker().run(id, [id, &db, keyspace, ranges = std::move(ranges), neighbors = std::move(neighbors)] () mutable {
             auto cfs = list_column_families(db.local(), keyspace);
+            if (cfs.empty()) {
+                rlogger.warn("repair id {} to sync data for keyspace={}, no table in this keyspace", id, keyspace);
```

Comment:
Why is it a warning?
If this can happen under normal circumstances then info or even debug should be enough.

Reply:
In normal cases, user would probably create at least one table when creating the keyspace. I put it as warn level because I worry the case that the keyspace has zero table because the table was dropped in the middle of the operation, to give the user a warning that this happens. It is up to the user to to decided.

Reply:
But the user may do that, right?
Do we recommend against it or try to prevent it in any way?

Reply:
Yes, user may do that. The recommendation is not to perform such operation during the topology changes.

Reply:
If we recommend against it than a warning is justified. thanks.